roles/openshift_common/tasks/atomic_docker_check.yml

```diff
@@ -0,0 +1,16 @@
+---
+# Fail early if Atomic and old version of Docker
+- block:
+
+  - name: Determine available Docker
+    shell: "rpm -q --queryformat '---\ncurr_version: %{VERSION}\n' docker"
+    register: l_atomic_docker_version_result
+
+  - set_fact:
+      l_atomic_docker_version: "{{ l_atomic_docker_version_result.stdout | from_yaml }}"
+
+  - fail:
+      msg: Installation on Atomic Host requires Docker 1.12 or later. Please upgrade and restart the Atomic Host.
+    when: l_atomic_docker_version.curr_version | version_compare('1.12','<')
+
+  when: openshift.common.is_atomic | bool
```

Comment:
That's a good question. The bug this is for is specifically for failing on AH since a reboot is required. Currently it looks like the installer will 

**File**: ./roles/docker/tasks/main.yml
```yaml
- name: Error out if Docker pre-installed but too old
  fail:
    msg: "Docker {{ curr_docker_version.stdout }} is installed, but >= 1.9.1 is required."
  when: not curr_docker_version | skipped and curr_docker_version.stdout != '' and curr_docker_version.stdout | version_compare('1.9.1', '<') and not docker_version is defined
```

I'm happy to bump that up to ``1.12`` in another PR.